.changelog/17251.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:new-resource
+aws_sagemaker_app
+```
+
+```release-note:enhancement
+resource/aws_sagemaker_user_profile: Fix destory check in acceptance tests
+```
```

Comment:
I don't think we need to call out acceptance test enhancements in the CHANGELOG.

Reply:
Not sure why i added it. Removed.